plugins/package-managers/node/src/main/kotlin/yarn/Yarn.kt

```diff
@@ -105,7 +107,8 @@ class Yarn(override val descriptor: PluginDescriptor = YarnFactory.descriptor) :
         val workspaceModuleDirs = getWorkspaceModuleDirs(workingDir)
         handler.setContext(workingDir, getModuleDirs(workingDir), workspaceModuleDirs)
 
-        val moduleInfosForScope = Scope.entries.associateWith { scope ->
+        val scopes = Scope.entries
```

Comment:
Personally, I'd have a slight preferences for keeping this inlined for clarity.

Reply:
Also here, I wanted to avoid repetition. So, I'd prefer slightly to keep this. 
(I'm also planning to filter excluded scopes in an upcoming PR)

Reply:
> Also here, I wanted to avoid repetition.

Just to double-check than I'm not missing something: The only "repetition" would be to write `Scope.entries` instead of `scope`, right? (Which to me is not really repetition, but slightly more verbose syntax for the same thing.)

> I'm also planning to filter excluded scopes in an upcoming PR in that line

Ok, *that's* the important information for me to get. So could you add to the commit message something like "This prepares for filtering excluded scopes in an upcoming change."? Then this is fine with me.